Mathlib/Tactic/CC/Datatypes.lean

```diff
@@ -69,19 +69,19 @@ scoped instance : Ord Expr where
 
 /-- Red-black maps whose keys are `Expr`s.
 
-TODO: the choice between `RBMap` and `HashMap` is not obvious:
+TODO: the choice between `TreeMap` and `HashMap` is not obvious:
 the current version follows the Lean 3 C++ implementation.
```

Comment:
This comment is no longer true right? The Lean 3 version was balanced by the red-black invariant, but these are size balanced instead